source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs

```diff
@@ -72,6 +100,22 @@ public TabBaseViewModel()
                 RaisePropertyChanged(() => Point2Formatted);
             });
 
+            // Kick off monitoring of mouse position via a timer
+            System.Timers.Timer mouseCheckTimer = new System.Timers.Timer();
```

Comment:
@rjones0 Your logic for getting mouse position is solid but I'm uncertain if using a timer is best in this situation. Please look at accepted anwsers in these links for possible workarounds: http://gis.stackexchange.com/questions/164739/handling-arcmap-closing-event-in-addin-to-save-edits-automatically and http://stackoverflow.com/questions/6036918/how-to-forward-messages-eg-mouse-wheel-to-another-control-without-stealing-fo.